README.md

```diff
@@ -1,32 +1,52 @@
+# Human Model Generator (HMG) [![License](https://img.shields.io/badge/License-BSD_3--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause) 
+
+The Human Model Generator is a Python-based tool designed to generate anthropometric human whole-body models in the Unified Robot Description Format (URDF) standard, suitable in robotics for motion analysis and simulation applications.
+
+## Introduction
+Dynamics is a critical aspect in the study of human movement, encompassing the forces and torques that cause motion. The generation of movement is possible through the synergistic action of the nervous system and muscles. The nervous system sends signals to the muscles, which then contract and produce movement by exerting forces on the skeletal system. The intricate interaction between the skeletal system and muscles allows for complex and coordinated motions.
+The musculoskeletal models offer a valuable approximation of the human anatomy and they are used extensively in fields like biomechanics, robotics, and computer simulations to study and mimic human movement. The muscle modeling is essential for a comprehensive understanding of the human movement. This involves the simulation of the behavior of muscles, including their activation patterns, force production, and interaction with the skeletal system. Accurate muscle models can predict how the whole-body movement and performance are affected by changes in muscle strength, coordination, and fatigue.  However, for a more accurate representation, additional data is required to customize models for different subjects, e.g., inertial parameters of body segments and anthropometric measurements.
+Therefore, the goal of the HMG is to develop an advanced human model that integrates both skeletal and muscular information. This model does not only include detailed anatomical and inertial data but also it is scalable to accommodate the specific features of each human subject.
+
+<br>
 
 <p align="center">
-<img src= https://github.com/ami-iit/human-model-generator/assets/116801366/ffb6bbd9-632a-4201-b7ee-dcb99a23fda2 width ="600" height="500">
+<img src= https://github.com/ami-iit/human-model-generator/assets/118193358/095c6a28-5dd8-4a73-a1ab-d2d5446c2e39 width ="650" height="500">
 </p>
 
-## Dependencies
-- [`python3`](https://wiki.python.org/moin/BeginnersGuide)
-- [`urdf-modifiers`](https://github.com/icub-tech-iit/urdf-modifiers)
+</br>
+
+## Dependencies 
+This library requires the following dependencies:
+
+- [``urdf-modifiers``](https://github.com/icub-tech-iit/urdf-modifiers)
+- [``idyntree``](https://github.com/robotology/idyntree)
+
+## Installation with [conda](https://docs.conda.io/en/latest/) (recommended)
+
+- Create and activate a brand new enviroment
+```
+conda create -n human_model_env
+conda activate human_model_env
+```
+- Install `urdf-modifiers`
```

Comment:
This installation has to be done after installing iDynTree otherwise  `pip` is not recognised. Please swap instructions (and to be consistent also swap order in lines 21 and 22)

Reply:
Done